.buildkite/pipeline-for-tags.yml

```diff
@@ -15,6 +15,10 @@ steps:
 
  - wait
 
+ - label: Add Mojave bottle hashes to formulae
```

Comment:
I assume that this job won't have access to the bottles by default to calculate hashes, you'll have to download them as artifacts from the previous job using
```
buildkite-agent artifact download --step build-bottles "*"
```
See the `Attach bottles to the release` job